content/posts/2025-vision.md

```diff
@@ -0,0 +1,97 @@
+* You can [override the ``build.jobs``](https://docs.readthedocs.io/en/stable/config-file/v2.html#build-jobs) to fully customize the build process.
+* If you don't want to use our build process at all, you can [run a fully custom build](https://docs.readthedocs.io/en/stable/build-customization.html#override-the-build-process) with ``build.commands``, allowing you to bring an existing build process to Read the Docs.
+
+This gives users an on-ramp into a simple build process,
+but allows advanced users to fully customize their build process as needed.
+
+## Addons make reading docs even better
+
+Addons give you a set of features that are enabled by default,
+but you can always choose to extend them or replace them fully.
+
+We're excited about the Addons we've already built:
+
+* [Visual diff](https://docs.readthedocs.io/en/stable/visual-diff.html) to see changes between versions of your documentation in the rendered HTML. This builds on top of [pull request previews](https://docs.readthedocs.io/en/stable/pull-requests.html) to give you a full suite of tools for reviewing documentation changes.
+* [Link previews](https://docs.readthedocs.io/en/stable/link-previews.html) to make browsing your docs easier for users, and improving the UX of link-heavy documentation like API references.
+* [Flyout menu](https://docs.readthedocs.io/en/stable/flyout-menu.html) that provides users access to many Read the Docs features like search, versions, offline formats, and more.
+* [Search as you type](https://docs.readthedocs.io/en/stable/server-side-search/index.html) that is powered by [Elasticsearch](https://www.elastic.co/elasticsearch/) and allows searching multiple subprojects at once.
+* [Traffic analytics](https://docs.readthedocs.io/en/stable/traffic-analytics.html) to understand how your documentation is being used, and where you can improve it.
+* [Notifications](https://docs.readthedocs.io/en/stable/doc-notifications.html) to alert users to out-of-date documentation, and other important information about their documentation builds.
+
+We have some more interesting ideas for additional Addons, like a [Command Palette](https://github.com/readthedocs/addons/pull/449) to expose all of the features of Read the Docs in a single searchable interface.
+
+### Integrate our data for a better UX
+
+With our [custom event integration](https://docs.readthedocs.io/en/stable/flyout-menu.html#custom-event-integration),
+you can always integrate the data provided by Read the Docs into your documentation.
+
+Some neat examples of what folks have been building:
+
+* Version selectors that use the API to show the latest versions of your documentation.
+* Offline formats that use the API to show the latest PDF or Epub downloads.
+* Search integrations that use the API to power a custom search interface.
+
+You can see an [integration example](https://test-builds.readthedocs.io/en/latest/) that shows how to use the API to build a custom version selector.
```

Comment:
I think you may have wanted to link to a difference place instead of test-builds?